docs/build/Build-Script.md

```diff
@@ -48,76 +48,91 @@ If given path is not exist, build.py will create it.
 ```
 
 --
-#### `--clean`
-With given this option, build.py will clear all the build directory before start new build.
+#### `--buildlib`
+With given this option, build.py will generate IoT.js output as a library.
 
 ```
-./tools/build.py --clean
+./tools/build.py ---buildlib
 ```
 
 --
-#### `--buildlib`
-With given this option, build.py will generate IoT.js output as a library.
+#### `--cmake-param`
+Specify CMake parameters for IoT.js.
+
+"cmake" command for IoT.js will be executed with the given parameter applied.
+
+If you have multiple parameters, supply it with multiple use of this option;
 
 ```
-./tools/build.py ---buildlib
+./tools/build.py --cmake-param="..." --cmake-param="..."
 ```
 
 --
-#### `--profile`
-With given this option, build.py will use the specified profile for the build.
+#### `--compile-flag`
+Specify C compiler flags for IoT.js.
 
+If you have multiple compile flags, supply it with multiple use of this option;
 ```
-./tools/build.py --profile=./profiles/minimal.profile
+./tools/build.py --compile-flag="..." --compile-flag="..."
 ```
 
 --
-#### `--target-arch`
-* `arm` | `x86` | `i686` | `x86_64` | `x64`
-
-Specify target architecture.
+#### `--clean`
+With given this option, build.py will clear all the build directory before start new build.
 
 ```
-./tools/build.py --target-arch=arm
+./tools/build.py --clean
 ```
---
-#### `--target-os`
-* `linux` | `darwin` | `osx` | `nuttx`
 
-Specify target OS.
+--
```

Comment:
Seemingly `--` in the previous doc was intended for `---`. Why don't we fix them this time?